### PR TITLE
Recyclerview functionality commit #4

### PR DESCRIPTION
**Pull request by kieranmiguel**
Recyclerview is now working with dummy data entered on main activity

### CODE REVIEW

**Review by Dylan-Riley — APPROVED** [review]
Looks like it fixes the issues I was having

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -14,16 +14,15 @@
...
         android:background="@color/BankGreen"
         android:id="@+id/toolbar"
-
         >
 
 
         <ImageView
-            android:layout_width="415dp"
+            android:layout_width="433dp"
             android:paddingVertical="10dp"
```
Comment (jleeboyd) [inline app/src/main/res/layout/activity_main.xml:22]: Probably want to extract these dimensions

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -51,11 +50,11 @@
     <ImageButton
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="600dp"
-        android:layout_marginLeft="80dp"
+        android:layout_marginTop="500dp"
+        android:layout_marginLeft="95dp"
         android:padding="0dp"
```
Comment (jleeboyd) [inline app/src/main/res/layout/activity_main.xml:55]: Here too.

**Review by jleeboyd — APPROVED** [review]
Looks good, and works on my machine.

I have one note regarding the drawables used. I'm not sure if these are just 'dummy' images, or if they are the ones we plan to use for the actual app. If so we may need to look into higher resolutions if possible. The images for Accounts and the Family Bank title are a little blurry on my end.

Screen Shot 2020-10-12 at 7 21 11 PM

**Pull request merged**